lib/lynx/system_out.rb

```diff
@@ -0,0 +1,13 @@
+require 'open3'
+require 'lynx/error'
+
+module Lynx
+  module SystemOut
+    def self.system_out_with_err(command)
+      stdout, stderr, status = Open3.capture3(command)
+      unless status.success?
+        raise(Lynx::Error, "\n [ERROR] #{stderr} \n [ERROR] Failed to perform: #{command}")
+      end
```

Comment:
Might be worth returning `stdout` so that https://github.com/panthomakos/lynx/blob/master/lib/lynx/pipe/get.rb can also use this?